CMakeLists.txt

```diff
@@ -29,10 +29,14 @@ target_link_libraries(effil -lpthread ${LUA_LIBRARY} -ldl)
 set_target_properties(effil PROPERTIES SUFFIX .so)
```

Comment:
Should it be `.dll` instead of `.so`?

Reply:
Yes. I will leave it to another PR as mentioned in the description.